Module 6/assignment5.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Assignment 5</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.css" />
+    <script src="http://code.jquery.com/jquery-1.11.1.min.js"></script>
+    <script src="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.js"></script>
+    <link rel="stylesheet" type="text/css" href="_css/bistro.css">
+</head>
+<body>
+<!-- Randall Mowen
+     Assignment5.html
+     INFO2439
+     Thoendel
+     10/15/19
+-->
+
+<div data-role="page" id="assignment4"> <!-- opening page div -->
+    <header>
+    <div data-role="header" data-position="fixed" data-id="header" class="top-header" data-theme="a"><!-- opening header div -->
+        <a href="../index.html" data-icon="home" rel="external">Home</a>
+        <h1>Randall Mowen</h1>
+        </div><!-- closing header div -->
+    </header>
+    
+    <div data-role="main" class="ui-content">
+    <h1>Assignment #5, Sage Bistro Static Google Map</h1>
+    <div class="ui-grid-a">
+        <div class="ui-block-a">
+            <img src="https://maps.googleapis.com/maps/api/staticmap?center=41.304729,-95.959506&zoom=12&size=150x150&scale=2&zoom=17&maptype=roadmap&key=AIzaSyAFQid4GkVK5cIqrk3n8kuRkAo7YcNeTzo" alt="Google map">
```

Comment:
Thank you for including a screenshot of your own API key in your BB submission! Perfecto!